README.md

```diff
@@ -80,9 +79,10 @@ staticAnalysis {
     findBugsExcludeFilterFile
 
     lintAbortOnError        // default:  false
-    lintConfigFile
 }
 ```
+<b>Please note that if you want to change lint's `abortOnError` setting that you'll need to do it via this value, since it will overwrite any value you put directly into the `lintOptions` block of your build.gradle file.</b>
```

Comment:
make this note more general than just abortOnError since it applies to all the settings here